foreman-selinux-enable

```diff
@@ -25,6 +29,10 @@ do
     /usr/sbin/semanage port -E | grep -q docker_port_t || \
       echo "port -a -t docker_port_t -p tcp 2375-2376" >> $TMP
 
+    if is_redhat_6; then
+      echo "port -a -t foreman_osapi_compute_port_t -p tcp 8774" >> $TMP
```

Comment:
The same `semanage port -E` check is needed here to skip the step if it's already defined (on EL6), else running this script multiple times will result in errors:

<pre>
# /usr/sbin/foreman-selinux-enable
/usr/sbin/semanage: Port tcp/8774 already defined
# echo $?
1
</pre>

The code I wanted removed was only the modification stuff.